databases/week2/school-db.sql

```diff
@@ -0,0 +1,37 @@
+USE `school-db`;
+
+CREATE TABLE `class` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `begin` DATETIME,
+  `ends`DATETIME NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+CREATE TABLE `student` (
+`id` int unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+  `class_id` int unsigned NOT NULL,
+  PRIMARY KEY (`id`)
+  -- CONSTRAINT `fk_class` FOREIGN KEY `class`(id) REFERENCES `student`(id) ON UPDATE CASCADE ON DELETE CASCADE
```

Comment:
Why did you comment this line? 
it's almost correct, but  the foreign key is class_id and it references the column id of the table class.